deel-1-flexbox-positioneren/styles/styles.css

```diff
@@ -1 +1,92 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+/
+*Deel 1
+Opdracht 1*/
```

Comment:
Voor commentaar is het belangrijk dat de slash en het sterretje aan elkaar zitten dus /* en geen regel of spatie er tussen.